app/parsing_scripts/rsync_operations.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+import os
+import json
+
+RSYNC = '/usr/bin/rsync'
```

Comment:
https://github.com/macports/macports-webapp/pull/33#discussion_r302761873

Reply:
You now use a variable, but these variables should be defined together in some "common settings" file.